src/schemathesis/cli/commands/__init__.py

```diff
@@ -16,6 +17,11 @@
 from schemathesis.core.errors import HookError, format_exception
 from schemathesis.core.version import SCHEMATHESIS_VERSION
 
+if sys.version_info < (3, 11):
+    from tomli import TOMLDecodeError
```

Comment:
The original import is still there. See line 8 above, and it will fail on Python >=3.11 as `tomli` won't be installed